.circleci/config.yml

```diff
@@ -23,6 +23,9 @@ localCheckout: &localCheckout
     #- run:
     #    name: Ubuntu-Bionic
     #    command: cd ubuntu-bionic && docker build --build-arg ARCH=x86_64 -t ci-ubuntu-bionic . && docker tag ci-ubuntu-bionic $TARGETNAME/ci-ubuntu-bionic-x86_64
+    - run:
+        name: Ubuntu-Bionic-i386
+        command: cd ubuntu-bionic && docker build --build-arg ARCH=i386 -t ci-ubuntu-bionic . && docker tag ci-ubuntu-bionic $TARGETNAME/ci-ubuntu-bionic-i386
```

Comment:
Why bionic (18)? Does LTS focal (20) cause problems with i386?

Reply:
Ubuntu stopped supporting 32-bit with Focal; Bionic is the last 32-bit Ubuntu LTS.

Reply:
OK - didn't know that. Thanks.